docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.9'
+services:
+  # Local development
+  database:
+    # image: postgres:15-alpine
+    image: postgres
+    restart: always
+    container_name: postgres
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=shortener-db
+    ports:
+      - '5433:5432'
+    volumes:
+      - ./.db/data:/var/lib/postgresql/data:delegated
+      - ./.db/init:/docker-entrypoint-initdb.d
+    healthcheck:
+      test: pg_isready -U postgres -h 127.0.0.1
+
+  pgadmin:
+    image: dpage/pgadmin4
+    container_name: pgadmin
+    environment:
+      - PGADMIN_DEFAULT_EMAIL=mail@villagesquare.com
+      - PGADMIN_DEFAULT_PASSWORD=password
+    ports:
+      - '5555:80'
```

Comment:
Why are we using this — `pgadmin`?

Reply:
There is no serious reason to have it there. It was merely to have pgadmin running in order to view that database. Thinking about it some more, we really don't need it.